### PR TITLE
Feat/set first connection with wps- processorF2

### PR DESCRIPTION
**Pull request by IoannaMi**
* Run processorF2
* Template for json request
* Add status card
* Re send request every 2 sec for 14 times
* Status change every time
* No success is received yet. Only failed.

### CODE REVIEW

File: config/services/services.json [hunk 1]
```diff
@@ -30,11 +30,13 @@
...
               ],
               "date_span": true,
-              "draggable_marker": true
+              "draggable_marker": {
+                "title": "Place marker at output location"
+              }
             }
           },
           {
-            "id": "fronts_detection",
```
Comment (IoannaMi) [inline config/services/services.json:57]: I would suggest to create a separate id for the wps name, like wps_id and leave the id as it was before. This id is more dedicated to the whole service.

File: src/components/ListJobs.vue [hunk 2]
```diff
@@ -0,0 +1,63 @@
+<template>
```
Comment (IoannaMi) [inline src/components/ListJobs.vue:1]: Can you create a folder list-jobs and move the component there? It would be nice consistency with the rest components

File: src/components/ListJobs.vue [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+  data () {
+    return {
+      jobs: []
+    }
+  },
+  mounted () {
+    this.getJobs()
+  },
+  methods: {
+    getJobs () {
```
Comment (IoannaMi) [inline src/components/ListJobs.vue:54]: Can you move the getJobx function in wps folder similar like the other functions. Then call this function in the component.  

Suggestion: While the function is loading the response there we can show a circle that is loading. I noticed that it is taking time to show the results.

File: src/components/service-viewer/service-viewer.vue [hunk 4]
```diff
@@ -84,6 +88,15 @@
       :firstStatus="jobStatus"
       :statusLink="statusLink"
     ></status-card>
+    <div v-if="service.components.joblist">
+    <collapsible-card
+      v-if="service.components.joblist"
+      :expand="0"
+      title="Overview of jobs"
+    >
+      <ListJobs :timeExtent="timeExtent"></ListJobs>
```
Comment (IoannaMi) [inline src/components/service-viewer/service-viewer.vue:114]: When you use the component write it as <list-jobs>

File: src/components/service-viewer/service-viewer.vue [hunk 5]
```diff
@@ -138,11 +152,13 @@ export default {
       timeSpan: (state) => state.timeSpan,
       jobStatus: (state) => state.jobStatus,
       selectedTime: (state) => state.selectedTime,
-      statusLink: (state) => state.statusLink
+      statusLink: (state) => state.statusLink,
+      selectedCategory: (state) => state.selectedCategory,
```
Comment (IoannaMi) [inline src/components/service-viewer/service-viewer.vue:175]: Why did you add them? They are not used in this component, so we can remove them? Or do you intend to use them?

File: src/lib/wps/runProcessor.js [hunk 6]
```diff
@@ -0,0 +1,28 @@
+import axios from "axios"
+import { template_f2 } from './template_f2'
+import { template_a2 } from './template_a2'
+
+
```
Comment (IoannaMi) [inline src/lib/wps/runProcessor.js:7]: Can you remove the comments?

File: src/components/ListJobs.vue (hunk 2 above)
Comment (danieltwigt) [reply src/components/ListJobs.vue:1]: done

File: src/lib/wps/runProcessor.js (hunk 6 above)
Comment (danieltwigt) [reply src/lib/wps/runProcessor.js:7]: done

File: config/services/services.json (hunk 1 above)
Comment (danieltwigt) [reply config/services/services.json:57]: Now we use the following state variable to store the service ID: state.selectedService. If we add a separate ID for the WPS process in the services.json (wps_id), should we also store this in a separate state variable (state.selectedWpsService) to keep track of this?

File: src/components/ListJobs.vue (hunk 3 above)
Comment (danieltwigt) [reply src/components/ListJobs.vue:54]: In general, when would you a place a function under src\lib\wps, and when would you include it in the Vue component directly (like I did here)? Is the following a good guideline: if it interacts with the WPS service, place it under src\lib\wps? Or would it be if we want to re-use code in multiple Vue components?

File: src/components/service-viewer/service-viewer.vue (hunk 4 above)
Comment (danieltwigt) [reply src/components/service-viewer/service-viewer.vue:114]: ? Not really clear what you mean with this one

Comment (IoannaMi) [reply src/components/service-viewer/service-viewer.vue:114]: as .. list-jobs :)

File: config/services/services.json (hunk 1 above)
Comment (IoannaMi) [reply config/services/services.json:57]: Yes you need to create a separate state for the wps_id
